x/marketmap/types/hooks.go

```diff
@@ -14,6 +14,9 @@ type MarketMapHooks interface {
 
 	// AfterMarketGenesis is called after x/marketmap init genesis.
 	AfterMarketGenesis(ctx sdk.Context, tickers map[string]Market) error
+
+	// AfterMarketRemoved is called after a market is removed.
+	AfterMarketRemoved(ctx sdk.Context, market Market) error
```

Comment:
If this hook errors, does the message fail?

Could we check if the removed market was enabled here, and just fail?

Reply:
Yes if these hooks fail, the message would fail